run.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
-JAVA_OPTS="-Djavax.net.ssl.trustStore=/app/trusted.jks -Djavax.net.ssl.trustStorePassword=${TRUST_STORE_PASSWORD} -Djavax.net.ssl.trustStoreType=jks"
+JAVA_OPTS="-Djavax.net.ssl.trustStore=/app/trusted.jks \
+           -Djavax.net.ssl.trustStorePassword=${TRUST_STORE_PASSWORD} \
+           -Djavax.net.ssl.trustStoreType=jks \
+           -Dcom.sun.management.jmxremote.local.only=false \
+           -Ddelius.endpoint.url=${DELIUS_ENDPOINT_URL} \
```

Comment:
don't think these two properties are needed, since they are set as environment variables instead

Reply:
Do you mean the DELIUS_* vars?

Reply:
As above, explicitly passing values from environment to JVM.